ros/src/waypoint_follower/src/pure_pursuit_core.cpp

```diff
@@ -234,9 +234,26 @@ bool PurePursuit::verifyFollowing() const
   double a = 0;
   double b = 0;
   double c = 0;
-  getLinearEquation(current_waypoints_.getWaypointPosition(1), current_waypoints_.getWaypointPosition(2), &a, &b, &c);
+  // edufford start
+  //getLinearEquation(current_waypoints_.getWaypointPosition(1), current_waypoints_.getWaypointPosition(2), &a, &b, &c);
```

Comment:
Remove commented out line - it will always be in github history where we can find it if we need it.